deploy/k8s/overlays/prod/kustomization.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: labelstudio
+
+namePrefix: labelstudio-
+
+bases:
+  - ../../base/
+
+resources:
+  - ingress.yml
+
+patchesStrategicMerge:
+  - env.yml
+
+commonLabels:
+  stage: production
+
+secretGenerator:
+- name: labelstudio
+  env: labelstudio.env
```

Comment:
Please add an empty file `labelstudio.env`

Reply:
well it can actually be even completely removed now that POSTGRE_PASS is removed